pkg/utils/k8sutil.go

```diff
@@ -0,0 +1,478 @@
+/*
```

Comment:
Use this pkg from `kubeless` as a package and create a different one here for the kafka related stuff

Reply:
For now, i used kubeless/kubeless repo. But eventually might be better to use a separate package as SDK. Vendoring github.com/kubeless/kubeless means vendoring lots of transitive dependencies as well.